locales/en.json

```diff
@@ -6,7 +6,7 @@
     "admin_password_changed": "The administration password got changed",
     "admin_password_too_long": "Please choose a password shorter than 127 characters",
     "already_up_to_date": "Nothing to do. Everything is already up-to-date.",
-    "app_action_cannot_be_ran_because_required_services_down": "This app requires some services which are currently down. Before continuing, you should try to restart the following services (and possibly investigate why they are down): {services}",
+    "app_action_cannot_be_ran_because_required_services_down": "Start all services required to run the action that failed. Try restarting the following ones (and possibly investigate why they are down): {services}",
```

Comment:
```suggestion
    "app_action_cannot_be_ran_because_required_services_down": "An action has failed because its required services were not running, {services}. Try restarting them (and possibly investigate why they are down).",
```

Reply:
Eeeh, the action did not already failed though ...

I dunno if that helps, but this string is used [here](https://github.com/YunoHost/yunohost/blob/stretch-unstable/src/yunohost/app.py#L3154) during a pre-action check [here](https://github.com/YunoHost/yunohost/blob/stretch-unstable/src/yunohost/app.py#L904) for example, prior to triggering the actual app install procedure, but it's already used prior to the app upgrade etc...